graphbolt/src/cuda/macro.h

```diff
@@ -0,0 +1,32 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file cuda/macro.h
+ * @brief CUDA macro definitions.
+ */
+#ifndef GRAPHBOLT_CUDA_MACRO_H_
+#define GRAPHBOLT_CUDA_MACRO_H_
+
+namespace graphbolt {
+
+/**
+ * @brief CUDA kernel launch macro.
+ *
+ * @param kernel Kernel function name.
+ * @param nblks Number of blocks.
+ * @param nthrs Number of threads per block.
+ * @param shmem Shared memory size.
+ * @param stream CUDA stream.
+ * @param ... Kernel arguments.
+ */
+#define GRAPHBOLT_CUDA_KERNEL_CALL(kernel, nblks, nthrs, shmem, stream, ...) \
+  {                                                                          \
+    (kernel)<<<(nblks), (nthrs), (shmem), (stream)>>>(__VA_ARGS__);          \
+    cudaError_t e = cudaGetLastError();                                      \
```

Comment:
You could use `C10_CUDA_KERNEL_LAUNCH_CHECK()` from [here](https://github.com/pytorch/pytorch/blob/3ca81aed4261f7bce3111c8905d979872f545f12/c10/cuda/CUDAException.h#L72).